src/databricks/labs/remorph/config.py

```diff
@@ -51,3 +55,27 @@ def get_write_dialect(self):
         if self.mode == "experimental":
             return _get_dialect("experimental")
         return _get_dialect("databricks")
+
+
+@dataclass
+class TableRecon:
+    source_schema: str
+    target_catalog: str
+    target_schema: str
+    tables: list[Table]
+    source_catalog: str | None = None
+
+
+@dataclass
+class DatabaseConfig:
+    source_schema: str
+    target_catalog: str
+    target_schema: str
+    source_catalog: str | None = None
+
+
+@dataclass
+class ReconcileOutput:
```

Comment:
let us move this inside the reconcile package only, since this is an internal dataclass

Reply:
Done.